### DISCUSSION TITLE
How can it be that battery devices go offline for periods during the day?

### DISCUSSION THREAD

**Discussion by eigenphase** [Koenkk/zigbee2mqtt#15435] [Devices]
I am using the Sonoff Zigbee P dongle and I am suffering a LOT from stability issues. I have created a strong mesh by placing mains powered plugs as routers every few meters. I do not think connectivity can really be an issue.

I recently got two ThirdReality blinds and from the beginning on, the went Unavailable/Offline at random times. This is a screenshot of the LQIs from Home Assistant over the past few days:

<img width="1793" alt="image" src="https://user-images.githubusercontent.com/7464149/206339658-f0b80c31-d378-4d22-8085-b862129c667e.png">

You can see many small places at which the line is discontinuous. A larger "hole" is for example Dec 5 at 6:30pm. This is the time when the devices go "Unavailable" in zigbee2mqtt and I can't control them any more for some time (until I manually hit a button on the device itself at which point they become Available again).

Note that the LQI is always in a fairly good range (around 120).

How on earth can this be?

For battery devices my understanding was that they just go offline if they don't send a beacon for 24h. So I would expect that they need to be offline for at least 24 hours to become unavailable.

How could I possibly fix this issue?

**Comment by grusso02** [comment]
Guarda non te lo so dire, chiama l'assistenza sangù
